src/lib/utils.js

```diff
@@ -808,6 +808,8 @@
       var mime = res.getResponseHeader('Content-Type').replace(/;.*/, '');
       ext = getFileExtensionFromMime(mime) || ext;
       return createFileEntryFromBlob(res.response, ext);
+    }).catch(function (res) {
+      throw new Error(res);
```

Comment:
I think this change is not necessary. Without rethrowing an error, promise status becomes an errored.